### PR TITLE
Display full property name for validation errors

### LINKED ISSUE #164: Display full property name for validation errors
**Is your feature request related to a problem? Please describe.**
As a user, it is hard to understand the validation errors since they do not show which schema the property is for.
- E.g. missing `schema_a.property_1` is displayed as `must have required property 'Property 1'` without mention that it is for `schema_a`.
- This gets confusing when there are many properties with the same name, which happens quite often with arrays.

Another example, seen in the screenshot below. The full property is present in the logs but not in the displayed UI,

image

**Describe the solution you'd like**
We can display the full property name by adding a `transformErrors` function to the rjsf props ([docs](https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/form-props#transformerrors)). Something like this would work since `error.stack` is what is displayed in the UI:
```
function transformErrors(errors) {
    return errors.map(error => {
      if (error.property) { error.stack = `${error.property}: ${error.stack}` }
      return error
    })
  }
```
**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by micahwoodard**
fixes #164 
- added transformErrors parameter
-  added function that displays property name

### CODE REVIEW

**Comment by micahwoodard** [comment]
Hi @helen-m-lin do I need to create unit tests for this and do I need to merge into main or dev?

**Comment by helen-m-lin** [comment]
> Hi @helen-m-lin do I need to create unit tests for this and do I need to merge into main or dev?

Yes, please see if you need to add or update the tests in `RenderForm.test.js`. All PRs should be for dev!  Once approved, the changes will be deployed first to https://metadata-entry.allenneuraldynamics-test.org/. We handle deploying to prod separately.

File: src/components/RenderForm.js [hunk 1]
```diff
@@ -70,6 +70,13 @@ function RenderForm (props) {
     saveToJSONFile(event.formData, schemaType)
   }
 
+  function transformErrors (errors) {
```
Comment (helen-m-lin) [inline src/components/RenderForm.js:73]: nit: add JSDocs for the new function, e.g.:
```
  /**
   * Transforms errors to prepend property name to error message
   * @param {object[]} errors List of errors from JSON Schema validation
   * @returns {object[]} Transformed errors
   */
```

**Review by helen-m-lin — APPROVED** [review]
Looks good. Thanks Micah!

**Pull request merged**